### PR TITLE
The check for response definition has empty body and empty bodyFIle

### PR DESCRIPTION
**Pull request by okondrashin**
1. Added simple test for situation when response definition has empty body and empty bodyFIle.
The check is moved to the beginning of the method.

2. Make code more readable.

### CODE REVIEW

File: src/test/java/com/opentable/extension/BodyTransformerTest.java [hunk 1]
```diff
@@ -18,13 +18,7 @@
...
 import org.junit.Test;
 
-import static com.github.tomakehurst.wiremock.client.WireMock.urlMatching;
-import static com.github.tomakehurst.wiremock.client.WireMock.aResponse;
-import static com.github.tomakehurst.wiremock.client.WireMock.post;
-import static com.github.tomakehurst.wiremock.client.WireMock.get;
-import static com.github.tomakehurst.wiremock.client.WireMock.postRequestedFor;
-import static com.github.tomakehurst.wiremock.client.WireMock.getRequestedFor;
-import static com.github.tomakehurst.wiremock.client.WireMock.urlEqualTo;
+import static com.github.tomakehurst.wiremock.client.WireMock.*;
```
Comment (tranhungt) [inline src/test/java/com/opentable/extension/BodyTransformerTest.java]: We prefer to have explicit imports for clarity and consistency, could you re-expand them?

**Comment by tranhungt** [comment]
@okondrashin Looks great! Thanks for the test and refactor. Just a tiny comment in the code, if you could address that.
I can merge and probably publish today.

File: src/test/java/com/opentable/extension/BodyTransformerTest.java (hunk 1 above)
Comment (tranhungt) [inline src/test/java/com/opentable/extension/BodyTransformerTest.java]: If you're using intellij.. https://stackoverflow.com/questions/3348816/intellij-never-use-wildcard-imports

Comment (okondrashin) [inline src/test/java/com/opentable/extension/BodyTransformerTest.java]: done :)

**Pull request merged**